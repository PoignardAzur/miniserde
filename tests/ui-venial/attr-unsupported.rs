use miniserde_derive_venial::Serialize;

#[derive(Serialize)]
struct Struct {
    #[serde(skip)]
    x: i32,
}

fn main() {}
