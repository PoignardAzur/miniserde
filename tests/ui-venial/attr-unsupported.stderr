error: unsupported attribute
 --> tests/ui-venial/attr-unsupported.rs:5:13
  |
5 |     #[serde(skip)]
  |             ^^^^
